### PR TITLE
Check analitycs and add new cards

### LINKED ISSUE #574: Analytics not working

### LINKED ISSUE #576: sitemap xml

### LINKED ISSUE #578: Chain Limits
Add chain limits to homepage stats

image

### LINKED ISSUE #582: Social media preview image
@tetogomez please have a look at this tool that tests social media images , it seems we have several things to fix

https://socialsharepreview.com/?url=https://mainnet.eosio.online

### PR DESCRIPTION
**Pull request by tetogomez**
### What does this PR do?

- Resolve #582 
- Resolve #578 
- Resolve #576 
- Resolve #574
- Replace ${transaction} by (transaction)

### Steps to test

1. first step
2. next step

#### CheckList

- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: webapp/public/index.html [hunk 1]
```diff
@@ -6,12 +6,38 @@
...
+    <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="csrf_token" content="" />
+    <meta property="type" content="website" />
+    <meta property="url" content="%PUBLIC_URL%/" />
+    <meta name="theme-color" content="#ffffff" />
+    <meta name="_token" content="" />
+    <meta name="robots" content="noodp" />
+    <meta
+      name="title"
+      content="EOSCR Network and Infrastructure Dashboard for EOSIO networks"
```
Comment (xavier506) [inline webapp/public/index.html:18]: Can we change this for `Network Monitor and Infrastructure Dashboard for EOSIO blockchain networks`

En vez de EOSIO se podråe poner el nombre de la red para cada instancia?

File: webapp/public/index.html [hunk 2]
```diff
@@ -6,12 +6,38 @@
...
+    />
     <meta
-      name="Description"
+      name="description"
       content="Dashboard for EOSCR Network and Infrastructure networks"
```
Comment (xavier506) [inline webapp/public/index.html:22]: The diescription should be different than the title

Comment (xavier506) [reply webapp/public/index.html:22]:
```
The dashboard is an open source project for monitoring network data and block producer infrastructure for EOSIO blockchains.
```

File: webapp/public/index.html (hunk 1 above)
Comment (tetogomez) [reply webapp/public/index.html:18]: done!

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**